html/views/overlay/index.js

```diff
@@ -38,18 +39,27 @@ function initialize() {
 				}
 				ensureSkaterExists(k.Skater, k.Team);
 
-				if(k.field == 'Number')	{ updateSkaterNumber(me,mb,v); }
-				if(k.field == 'Flags') {
+				if (k.field == 'Flags') {
 					$('.'+k.field, me).attr('data-flag', v);
 					$(mb).attr('data-flag', v);
-				} else if (v == "") {
-					c = $('.'+k.field, me).html('&nbsp;');
```

Comment:
I think these assignments to undeclared variable c are extraneous, but if there's a one-character global variable floating around triggering side effects then I could have introduced a bug here?

Reply:
There shouldn't be. This code use to be all cryptic variables names, and most likely I missed this back when I was untangling it.